tests/test_strategies.py

```diff
@@ -277,3 +277,38 @@ def test_only_check_game_end_once(self):
         player.tree_search()
         # check that we didn't visit the pass node any more times.
         self.assertEqual(player.root.child_N[pass_move], 1)
+
```

Comment:
these tests look great and i'm happy to LGTM right now.  I think some tests for selfplay_mcts.py would also be useful to make sure the should_resign and set_result calls are happening correctly.